public/tracker/vaccine_table.csv

```diff
@@ -1,5 +1,6 @@
 No Vaccine,1.00,1.00,1.00
 "Unknown vaccine, unknown date",0.40,0.40,0.40
-Pfizer,1.00,0.56,0.20
-Moderna,1.00,0.56,0.20
+Pfizer,1.00,0.56,0.10
+Moderna,1.00,0.56,0.10
 AstraZenica,1.00,0.56,0.40
+Johnson&Johnson,1.00,0.33
```

Comment:
Can you add spaces before and after the ampersand?

Reply:
Fixed it by just adding a 2nd dose number.